src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -108,34 +126,36 @@ def get_zipcode_map(
         .groupby(["state", "puma"])
     )
 
-    for (state_id, puma), df_locale in simulation_addresses.groupby(["state_id", "puma"]):
+    for (state_id, puma), df_locale in simulation_addresses.groupby(groupby_cols):
```

Comment:
The groupby columns are always the output columns minus the column name. It'd be cleaner to define them here like that rather than defining a variable up top.

Reply:
Yes and I will use this in the other necessary places that it should be used.